lib/packages/core/lib/src/views/home/home_feed.dart

```diff
@@ -129,6 +130,26 @@ class _LMChatHomeScreenState extends State<LMChatHomeScreen> {
     return AnnotatedRegion<SystemUiOverlayStyle>(
       value: SystemUiOverlayStyle.dark,
       child: Scaffold(
+        floatingActionButton: FloatingActionButton(
+          onPressed: () async {
+            debugPrint("Floating action button pressed");
```

Comment:
remove debugPrint()

Reply:
Was this code just for testing? As I see all of this removed in the next PR anyway.

Reply:
yes it was for testing and removed in next PR